public/_redirects

```diff
@@ -1,2 +1 @@
-/ipns/* /?helia-sw=/ipns/:splat 302
-/ipfs/* /?helia-sw=/ipfs/:splat 302
+/* /?helia-sw=/:splat 302
```

Comment:
we should be able to remove this and just resolve https://github.com/ipfs-shipyard/helia-service-worker-gateway/issues/28